Pilot/pilotCommands.py

```diff
@@ -1210,3 +1210,33 @@ def execute( self ):
     """
     self._setNagiosOptions()
     self._runNagiosProbes()
+
+class UnpackDev( CommandBase ):
+  """ Unpack dev.tgz from the pilot directory into the pilot directory
+      Put dev.tgz in the remote pilot directory to have it fetched along
+      with the rest of the pilot scripts. The UnpackDev pilot command
+      needs to be listed after InstallDIRAC if it contains dev versions
+      of DIRAC modules.
+  """
+
+  def __init__( self, pilotParams ):
+    """ c'tor
+    """
+    super( UnpackDev, self ).__init__( pilotParams )
+    self.devFile = 'dev.tgz'
+
+  def execute( self ):
+    """ Standard entry point to a pilot command
+    """
+    self.log.info( 'Adding %s to bashrc' % os.getcwd() )
+    retCode, output = self.executeAndGetOutput( 'sed -i "s,^PYTHONPATH=,PYTHONPATH=%s:," bashrc' % os.getcwd() )
+    self.log.info( output, header = False )
+            
+    self.log.info( 'Unpacking ' + self.devFile )
+    retCode, output = self.executeAndGetOutput( 'tar zxvf ' + self.devFile )
```

Comment:
I think using `tarfile` https://docs.python.org/2/library/tarfile.html is much cleaner!